packages/dcos-calico/build

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -x
+
+# copy calico CNI plugin to CNI plugin directory
+mkdir -p $PKG_PATH/cni
+cp /pkg/src/calico/calico-amd64 $PKG_PATH/cni/calico
+chmod +x $PKG_PATH/cni/calico
+cp /pkg/src/calico-ipam/calico-ipam-amd64 $PKG_PATH/cni/calico-ipam
+chmod +x $PKG_PATH/cni/calico-ipam
+
+
+# create `dcos-calicoctl` to simply the usage of calicoctl by passing the
```

Comment:
We can benefit from `dcos-calicoctl` from devops and debugging inside the cluster. With the work of Pawel providing calicoctl function outside of the cluster, we will see if this is still useful.